project/plugins.sbt

```diff
@@ -11,3 +11,4 @@ addSbtPlugin("org.scala-js" % "sbt-scalajs" % scalaJSVersion)
 addSbtPlugin("org.scala-native" % "sbt-scala-native" % scalaNativeVersion)
 addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.5.3")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.6.0")
+addSbtPlugin("ch.epfl.scala" % "sbt-version-policy" % "1.0.0-RC5")
```

Comment:
should we add this to https://github.com/scala/sbt-scala-module/blob/2.x/build.sbt instead?

Reply:
That was consider and discussed at https://github.com/scala/sbt-scala-module/issues/111 . I was on the fence about it, but in the end I just shrugged and made a decision not to do it.

In general, my experiences while doing community build work have given me a somewhat dim view of putting build stuff in plugins. It makes individual builds harder to read and harder to maintain. As soon as settings and tasks are coming from a plugin, you can't just read the build and see what's going on anymore. You have to consider the possibility that a bunch of random stuff is coming in from some plugin and then you have to go read the sources of the plugin, too.

And then maintenance is harder for individual module maintainers, because they can't easily make changes to sbt-scala-module themselves. (And it's already hard enough to find and retain module maintainers.)

Reply:
I suppose sbt-scala-module could add sbt-version-policy but not actually set the settings. That's a middle ground I'm not sure I considered.

But anyway, 🤷

Reply:
Note that now that we have the Scala Steward, we get version bumps that way, so there's less benefit in getting them via sbt-scala-module.